src/app/(auth)/login/(components)/LoginHeader/LoginHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { authOptions } from '@/app/(auth)/auth'
+import { findUserByEmail } from '@/backend/user/user.service'
+import { getServerSession } from 'next-auth'
+import styles from './LoginHeader.module.scss'
+
+// components
+import MyProfileBtn from '@/app/(profile)/(components)/MyProfileBtn/MyProfileBtn'
+import CreateProfileBtn from '@/app/(profile)/my-profile/(components)/CreateProfileBtn/CreateProfileBtn'
+import GithubStarsButton from '@/components/Button/GitHubStarsBtn'
+import { Container } from '@/components/Container/Container'
+import Logo from '@/components/Logo/Logo'
+
+const LoginHeader = async () => {
+  const session = await getServerSession(authOptions)
+
+  const user = session?.user?.email
```

Comment:
**nit:** I think we can make it reausable function that gets our DB user from session or returns null if not found. We have something similar in `app/(auth)/auth.ts` called `authorizeUser`. I think we can refactor this function, as Im not sure we use it anywhere

Reply:
implemented as `getAuthorizedUser()`